### PR TITLE
Update CI

### PR DESCRIPTION
**Pull request by dralley**

### CODE REVIEW

File: template_config.yml [hunk 1]
```diff
@@ -38,7 +38,7 @@ plugins:
 - app_label: file
   name: pulp_file
 - app_label: certguard
-  name: pulp-certguard
+  name: pulp_certguard
```
Comment (dralley) [inline template_config.yml:41]: ^

Comment (mdellweg) [reply template_config.yml:41]: Hmmm. Should be fine. After all the name will be canonicalized in the really important places anyway.
Maybe you want to rename the subdirectory too and we filanny have all the plugins follow the same naming convention.

Comment (mdellweg) [reply template_config.yml:41]: Oh, the directory has always been `pulp_certguard` only the repository was hyphened.

Comment (dralley) [reply template_config.yml:41]: Yes, the name of the file in the bumpconfig was correct (used underscore), and when it became templatized based on this value it became incorrect.

**Pull request merged**